lib/collectionspace/converter/publicart/config.yml

```diff
@@ -129,30 +129,41 @@ registered_profiles:
       Authorities:
         Organization:
         - currentowner
-  person:
+  organization:
     type: Authorities
     enabled: true
     identify_by_column: termdisplayname
     required_headers:
       - termdisplayname
     config:
-      mapper: PublicArtPerson
+      mapper: PublicArtOrganization
       name_field: termdisplayname
-      authority_type: Person
-      authority_subtype: person
-      Authorities: []
-      Vocabularies: []
-  organization:
+      authority_type: Organization
+      authority_subtype: organization
+      Authorities:
+        - name_field: currentplace
+          authority_type: Place
+          authority_subtype: place
+        - name_field: foundingplacelocal
+          authority_type: Place
+          authority_subtype: place
+        - name_field: foundingplaceshared
+          authority_type: Place
+          authority_subtype: place  
```

Comment:
@Paulmulonzia - Same as on the Person review I just did... The other place this form should be is in the converter code itself! Which is probably easiest! https://github.com/collectionspace/cspace-converter/blob/f5fee0fddc3da380375664706fc8194f74cb6949/lib/collectionspace/converter/publicart/organization.rb#L80

Reply:
@Paulmulonzia - Here's how I'd approach these updates, since unfortunately the config settings do not cascade down to publicart from core. 

For organization record: 
 - update core config.yml to make sure all authorities/vocabularies used by fields in core org record are added
 * copy the auth/vocab config section from core to publicart config.yml
 * in publicart config.yml:
 ** delete auth/vocabs for any fields in core but omitted from publicart
 ** add auth/vocabs for fields in publicart but not in core
 ** verify auth/vocabs for core fields overridden by publicart (delete or add auth/vocabs as needed)

Reply:
@kspurgin Ready for review.

Reply:
tracking this on #319 , proceeding to close this one.